src/GameRenderer.fs

```diff
@@ -124,21 +125,25 @@ let render (stateModel: StateModel) (renderModel: RenderModel option) dispatch (
       rmodel
 
     | LaunchTitle ->
-      let messages = [|"MERRY MERRY"; "CHRISTMAS"|]
+      let name = getParameterByName("name")
+      let messages =
+        if isNull name
+        then [|"MERRY CHRISTMAS"|]
+        else [|"MERRY CHRISTMAS"; name|]
       let space = 100. * rmodel.Scale
-      let maxSpace = messages |> Seq.map (fun m -> m.Length) |> Seq.max |> float |> (*) space
-      let xMargin = (gameWidth - maxSpace) * 0.5 + space * 0.5
 
       let textOptions = jsOptions<TextStyleOptions>( fun o ->
-        o.fill <- Some !![|"#19b14c";"#f4e182"|]
-        o.fontSize <- !!100.
-        o.fontFamily <- !!"Playfair Display SC"
+        o.fill <- Some(box [|"#19b14c";"#f4e182"|])
+        o.fontSize <- Some !^100.
+        o.fontFamily <- Some !^"Playfair Display SC"
```

Comment:
Not strictly necessary, but I've replaced `!!` with the `!^` operator as the latter gives you actual type checking :+1:

Reply:
Great. I always forget about thsi operator. Thanks it's now clean! 👍